src/InnerImageZoom/components/Image.js

```diff
@@ -18,9 +18,9 @@ const Image = ({ src, sources, width, height, hasSpacer, imgAttributes, isZoomed
               createSpacer ? 'iiz__img--abs' : ''
             }`}
             style={{
-              transition: `linear 0ms opacity ${isZoomed ? fadeDuration : 0}ms, linear ${fadeDuration}ms visibility ${
-                isZoomed ? fadeDuration : 0
-              }ms`
+              transition:
+                `opacity 0ms linear ${isZoomed ? fadeDuration : 0}ms, ` +
+                `visibility ${isZoomed ? 0 : fadeDuration}ms linear ${isZoomed ? fadeDuration : 0}ms`
```

Comment:
I really had to think back on the original intention for this image. The image should disappear after the zoom image has transitioned in without any effect on itself. It should reappear immediately before the zoom image begins transitioning out, again with no effect.

The overrun you were seeing looks like it was caused by the transition duration on visibility. So this line should probably look something like this instead:

```
`visibility 0ms linear ${isZoomed ? fadeDuration : 0}ms`
```